### PR TITLE
Update Readme

### PR DESCRIPTION
**Pull request by mhogerheijde**
- Some typo/language fixes
- Changed order of things a bit

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -1,100 +1,147 @@
...
+4. redirect request from bots to PhantomJS
+
+1. Modifying your static HTML
+-----------------------------
 
-Modifying your static HTML
-==========================
+The `#!` escape for browsers that use HTML5 push-state makes crawlers
+request your page as
+`http://your.site/page?_escape_fragment_=some/push/state/url. If you
```
Comment (SAGV) [inline readme.md]: I feel like that the paragraph is more confusing than helping

File: readme.md [hunk 2]
```diff
@@ -1,100 +1,147 @@
...
-PhantomJS Module
-================
+If Phantom was not called at all, the page will be rendered with 10
+seconds delay.
+
+3. PhantomJS Module
+-------------------
+
+For the app to be properly rendered, you will need to run the `angular-
+seo-server.js` with PhantomJS. Make sure to disable caching:
```
Comment (SAGV) [inline readme.md]: it's just `server.js`

File: readme.md [hunk 3]
```diff
@@ -1,100 +1,147 @@
...
-				proxy_pass  https://<phantomJS-Machine>/$uri?$args;
+		    # Only proyx if $prerender is not set to 0.
+			proxy_pass  http://localhost:8082/$uri?$args;
 		}
-}
+	}
 ```
+
+
+Check your setup
```
Comment (SAGV) [inline readme.md]: Somehow, github assumes that it's still a code block until end of the file

File: readme.md (hunk 2 above)
Comment (mhogerheijde) [inline readme.md]: ah sorry, I probably undid some of your changes! Will double check and revert this.

File: readme.md (hunk 1 above)
Comment (mhogerheijde) [inline readme.md]: Ok. I'll include a link to escape fragments, in stead of elaborating about it here.

The thing is, that this is only needed if your app uses push-state without `#!`,.. right?

File: readme.md (hunk 3 above)
Comment (mhogerheijde) [inline readme.md]: hmmm,.. strange.

**Comment by mhogerheijde** [comment]
@SAGV Fixed :)

File: readme.md [hunk 4]
```diff
@@ -1,100 +1,134 @@
...
-```
-$ phantomjs --disk-cache=no angular-seo-server.js [port] [URL prefix]
-```
+### 3. PhantomJS Module
 
-`URL prefix` is the URL that will be prepended to the path the crawlers will try to get.
+For the app to be properly rendered, you will need to run the
+`server.js` with PhantomJS. Make sure to disable caching:
+
+	$ phantomjs --disk-cache=no server.js [port] [URL prefix]
```
Comment (SAGV) [inline readme.md]: First url, then port :-)

Comment (mhogerheijde) [inline readme.md]: Whaa!! Fixed :)

**Comment by mhogerheijde** [comment]
Huzah!

**Pull request merged**